libs/component-library/src/lib/radio-button/RadioButton.tsx

```diff
@@ -0,0 +1,228 @@
+import { createBox, useTheme } from '@shopify/restyle';
+import * as React from 'react';
+import {
+  Animated,
+  View,
+  StyleSheet,
+  TouchableOpacity,
+  TouchableOpacityProps,
+} from 'react-native';
+import { FxTheme } from '../theme/theme';
+
+import type { $RemoveChildren } from './../types';
+
+import { useRadioButtonContext } from './RadioButtonGroup';
+import { handlePress, isChecked } from './utils';
+
+const TouchableBox = createBox<FxTheme, TouchableOpacityProps>(
```

Comment:
we have `FxPressableOpacity`, do we want to use that?

Reply:
Have updated to use.